test/Azure.Functions.Cli.Tests/ExtensionBundleHelperTests.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.IO;
+using Azure.Functions.Cli.Common;
+using Azure.Functions.Cli.ExtensionBundle;
+using Azure.Functions.Cli.Helpers;
+using Xunit;
+
+namespace Azure.Functions.Cli.Tests
+{
+    public class ExtensionBundleHelperTests
+    {
+        [Fact]
+        public void VerifyGetBundleDownloadPathReturnCorrectPath()
+        {
+            var downloadPath = ExtensionBundleHelper.GetBundleDownloadPath("BundleId");
+            var expectedPath = Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.UserProfile), ".azure-functions-core-tools", "Functions", "ExtensionBundles", "BundleId");
+            Assert.Equal(expectedPath, downloadPath);
```

Comment:
Can you update the test to verify path for each platform:

```
OSPlatform.Windows
OSPlatform.OSX
OSPlatform.Linux
```
You can use `System.Runtime.InteropServices.RuntimeInformation.IsOSPlatform()` to detemine the platform and verify with a hardcoded path.

Reply:
We can get the root path based on `System.Runtime.InteropServices.RuntimeInformation.IsOSPlatform()` but UserProfile would also include the user account used to execute the test. The only way to look that up would be by `Environment.SpecialFolder.UserProfile`